addons/glusterfs/dev-guide/glusterfs-zh_CN.md

```diff
@@ -0,0 +1,121 @@
+# GlusterFS 接入 ThinRuntime 的简单示例
+
+## 前期准备
+glusterfs 接入 ThinRuntime 需要构造 GlusterFS-FUSE 客户端，本实例使用 [该项目](https://github.com/gluster/gluster-containers) 为基础构建 FUSE 镜像。
+
+## 准备 GlusterFS-FUSE 客户端镜像
+1. 挂载远程文件系统脚本
+
+在 FUSE 容器内需要提取相关的 **ThinRuntimeProfile、Dataset、ThinRuntime**资源中对远程文件系统的配置信息，相关信息以 JSON 字符串的方式保存到 FUSE 容器的 **/etc/fluid/config.json** 文件内。
+
+```python
+import json
+import os
+import subprocess
+
+obj = json.load(open("/etc/fluid/config.json"))
+
+mount_point = obj["mounts"][0]["mountPoint"]
+target_path = obj["targetPath"]
+
+os.makedirs(target_path, exist_ok=True)
+args = ["mount", "-t", "glusterfs", mount_point, target_path]
+
+if "options" in obj["mounts"][0]:
+    options = obj["mounts"][0]["options"]
+    for option in options:
+        args.extend(["-o", f"{option[0]}={option[1]}"])
+
+subprocess.run(args)
+```
+该 shell 脚本创建挂载的文件夹并将远程文件系统挂载到目标位置（targetPath）。
+
+2. 创建 FUSE 客户端镜像
+
+
+将脚本打包入镜像。
+
+```dockerfile
+FROM gluster/glusterfs-client:latest
+
+ADD entrypoint.py /
+
+CMD ["bash", "-c", "python3 /entrypoint.py;sleep infinity"]
```

Comment:
Is possible to make clusterfs mount in foreground mode?  Seems there is `--no-daemon` mode?

https://manpages.ubuntu.com/manpages/trusty/en/man8/glusterfs.8.html

Reply:
Use `glusterfs with --no-daemon` instead of `mount` command.